docs/api/python/ndarray/ndarray.md

```diff
@@ -625,16 +571,54 @@ We summarize the interface for each class in the following sections.
     Custom
 ```
 
+## Linear Algebra
```

Comment:
put linalg/rand/contrib in separate pages like sparse

Reply:
ok.. I misunderstood your previous comment.